analyses/0012-TUF/implementation.md

```diff
@@ -0,0 +1,190 @@
+  - Document TUF installation procedures on a separate page (At the moment, it's
+    in the spec doc)
+  - Develop quick start and contribution guides for new users
+
+- **Content maintainability and creation process**
+  - Rename the website repo to a recognizable name e.g _TUF-ReadTheDocs_ or
+    similar
+  - Add search functionality to the website to make it easier to find content
+  - Identify maintainers for the website repo
+  - Add labels to the website repo
+  - Add a README to the website repo with information about the project
+    including links to important project repos
+  - Develop a contributors' guideline for new users
+  - Create procedural material for developing the website locally
+  - Include communication channels on the website repo README
+  - Provide information about project meetings e.g a meeting link and calendar
+    on both the website and repo
+  - Add the [TUF blog](https://theupdateframework.github.io/python-tuf) to the
+    website.
+
+## Implementation
+
+## Reorganize documentation
+
+### Align information in related sections
+
+Some sections listed on the menu bar have unrelated sub-sections. This structure
+makes information hard to find and can be confusing to new users. Much of the
+information on the website can go under a _Docs_ section. Consider the following
+structure:
+
+- **About**: Overview of TUF project
+- **Documentation**:
+
+  - Introduction to TUF
+  - TUF project components
+  - Getting started: TUF Specification, Standardization, reference
+    implementation, and documentation. You can include installation guides for
+    each component in this section
+  - Tutorials
+  - Implementations
+  - Best practices
+  - Contribution guidelines
+
+- **Community**: You can have two sections.
+
+  - Learn and connect: Includes all community communication channels including
+    social media, mailing lists, calendars, Slack, etc.
+  - Develop and Contribute: Information about how to contribute to TUF
+    components and documentation.
+  - Code of conduct
+
+- **Resources**:
+  - Blog
+  - News
+  - Videos
+- **Contact**:
+  - Maintainer info and TUF contact information
+
+### Consolidate some pages
+
+The specification index is referenced several times on the website despite
+having a sub-section.
+
+You can have a **docs section** with information tailored to the user roles of
+the three project components.
+
+### Add user roles to the Getting Started section
+
+Structure the _Getting Started_ section according to user roles under a _Docs_
+section. The perceived user roles for this project are:
+
+- **TUF specification**: Uses TUF metadata to download and verify targets
+- **TUF proposals contributor**: Proposes major changes to the specification,
+  including new features made as TUF Augmentation Proposals (TAPs)
+- **TUF implementation/Developers** : Develops software with the TUF Python
+  reference implementation
+- **TUF docs**: Contributes to documentation
```

Comment:
I would reframe those user roles a bit. What do you think about the following categories?

1. Interested person - Wants to learn more about TUF without specific agenda. 
1. Adopter - Wants to learn how they can use TUF.
   1. Repository maintainer
   1. Client maintainer 
1. Contributor - Wants to learn how they can engage with the community and contribute to the project

Reply:
> I would reframe those user roles a bit. What do you think about the following categories?
> 
> 1. Interested person - Wants to learn more about TUF without specific agenda.
> 2. Adopter - Wants to learn how they can use TUF.
>    
>    1. Repository maintainer
>    2. Client maintainer
> 3. Contributor - Wants to learn how they can engage with the community and contribute to the project

Makes sense. Thanks for identifying the personas